repositories/repositories.go

```diff
@@ -32,18 +80,21 @@ func NewQueryBuilder() squirrel.StatementBuilderType {
 	return squirrel.StatementBuilder.PlaceholderFormat(squirrel.Dollar)
 }
 
-func NewRepositories(
-	firebaseClient *auth.Client,
-	marbleConnectionPool *pgxpool.Pool,
-	metabase Metabase,
-	transfercheckEnrichmentBucket string,
-) Repositories {
-	executorGetter := NewExecutorGetter(marbleConnectionPool)
+func NewRepositories(opts ...Option) Repositories {
```

Comment:
Since the db pool seems to always be a necessary option, let's keep it as a mandatory param (and the optional ones like you defined here) ?